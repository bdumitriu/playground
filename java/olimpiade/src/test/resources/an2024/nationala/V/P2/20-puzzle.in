2 101 2
123 123 123 123 123 123 123 123 123 123 123 123 123 123 123 123 123 123 123 123 123 123 123 123 123 321 321 321 321 321 321 321 321 321 321 321 321 321 321 321 321 321 321 321 321 321 321 321 321 321 456 456 456 456 456 456 456 456 456 456 456 456 456 456 456 456 456 456 456 456 456 456 456 456 456 654 654 654 654 654 654 654 654 654 654 654 654 654 654 654 654 654 654 654 654 654 654 654 654 654 1234