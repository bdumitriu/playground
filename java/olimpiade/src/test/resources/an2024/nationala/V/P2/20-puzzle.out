1234 1231321
